module4/src/com/epam/aggregation/starter/TaskLauncher.java

```diff
@@ -0,0 +1,61 @@
+package com.epam.aggregation.starter;
+
+import com.epam.aggregation.entity.*;
+import com.epam.aggregation.logic.BankWorker;
+import com.epam.aggregation.logic.CarWorker;
+import com.epam.aggregation.logic.StateAggregator;
+import com.epam.aggregation.logic.TextAggregator;
+
+public class TaskLauncher {
+
+    public static void main(String[] args) {
+
+        // Task 1
+        Word word = new Word("Heading");
+        Word word2 = new Word("new Heading");
+        Sentence sentence = new Sentence("Body text");
+        Sentence sentence2 = new Sentence("Body text 2");
+        Sentence sentence3 = new Sentence("Body text 3");
+        Sentence sentence4 = new Sentence("Body text 4");
+        Text text = new Text(word, TextAggregator.createSentences(sentence,sentence2,sentence3));
+        TextAggregator.printText(text);
```

Comment:
You don't need really this class. Just create this list here or rename it like TextInitializer. 
 
In addition printText should be also in class - Text.